src/app/config/config.dev.json

```diff
@@ -1,5 +1,7 @@
 {
   "api-url": "http://localhost:8080",
   "api-version": "v2",
-  "app-url": "http://localhost:4200"
+  "app-url": "http://localhost:4200",
+  "tree-node-counts-update": true,
+  "autosave-subject-sets": false
```

Comment:
I would also put this new flag to the same place as tree-node-counts-update, i.e. in query.service constructor